### DISCUSSION TITLE
Adding a variable without a parent node/object.

### DISCUSSION THREAD

**Discussion by jeroenvdbergh** [FreeOpcUa/opcua-asyncio#1315] [Q&A]
Hi,

I'm using opcua-syncio to build simulators for projects. I need to add a variable to the server without a parent object. 

So it will show up right under the Objects folder in uaExpert instead of inside an Object. I already tried creating an Object without a name, but this still creates a parent object (with an empty name).

Based on the examples, it seems you can only add variables through objects, is there any other way to accomplish this?

Thanks in advance

**Comment by schroeder-** [comment]
This should work.
`await server.nodes.objects.add_variable(idx, "MyVariable", test_string)`
 But the Objects folder isn't names objects for an reason. So you either create a folder or object, to be complaint to the spec.
